File: Python/src/Problem001.py
__author__ = 'Joel'


def multiples():
    return sum([3 * x for x in range(1, 1000 // 3 + 1)]) + sum([5 * x for x in range(1, 1000 // 5)]) - sum(
        [15 * x for x in range(1, 1000 // 15 + 1)])


print(multiples())

File: Java/src/solutions/Problem001.java
package solutions;

import framework.Problem;
import framework.ProblemInfo;

@ProblemInfo(
        description = "Multiples of 3 and 5",
        number = 1
)
public class Problem001 extends Problem {
    @Override
    public void solve() {
        int sum = 0;
        for (int i = 0; i < 1000; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                sum += i;
            }
        }
        System.out.println(sum);
    }
}